provisioning/roles/wazuh/ansible-wazuh-agent/tasks/MacOS.yml

```diff
@@ -6,3 +6,48 @@
 
 - include_tasks: "installation_from_custom_packages.yml"
   when: wazuh_custom_packages_installation_agent_enabled
+
+- name: macOS | Installing agent configuration (ossec.conf)
+  template:
+    src: var-ossec-etc-ossec-agent.conf.j2
```

Comment:
macOS path is `/Library/Ossec` not `/var/ossec`. We should change the file name.

Reply:
It is the name of the template. It does not matter which name have. This is inherited from wazuh-ansible